Common/UnitDefinitions/Speed.json

```diff
@@ -30,7 +30,7 @@
     },
     {
       "SingularName": "MeterPerMinute",
-      "PluralName": "MetersPerMinutes",
+      "PluralName": "MetersPerMinute",
```

Comment:
Jeez, we had this many errors in plural form? I must have been sloppy in the reviews 😓 

Nice to get them corrected though!